src/include/simeng/Elf.hh

```diff
@@ -29,10 +29,16 @@ class Elf {
   uint64_t getProcessImageSize() const;
   bool isValid() const;
   uint64_t getEntryPoint() const;
+  uint64_t getPhdrTableAddress() const;
+  uint64_t getPHENT() const;
+  uint64_t getPHNUM() const;
 
  private:
   uint64_t entryPoint_;
-  std::vector<ElfHeader> headers_;
+  std::vector<Elf64_Phdr> pheaders_;
```

Comment:
Why `pheaders_`?

Reply:
According to the [man page](https://man7.org/linux/man-pages/man5/elf.5.html) the elf header contains information about program headers as well as section headers. Although sections headers are ignored, for clarity when referencing the documentation I changed the name.

               uint16_t      e_phentsize;
               uint16_t      e_phnum;
               uint16_t      e_shentsize;
               uint16_t      e_shnum;
               uint16_t      e_shstrndx;